### PR TITLE
SociLink

### PR DESCRIPTION
**Pull request by Queenen**
Thank you for your feedback on improvements and possible deficiencies. :)

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  top: 0.5rem;
+  width: 34%;
+}
+
+.create-post {
+  background-color: blue;
+}
+
+/* MEDIA QUERIES */
+@media screen and (min-width: 992px) {
```
Comment (sulenchy) [inline css/styles.css:118]: Use Bootstrap breakpoint for this

Comment (sulenchy) [reply css/styles.css:118]: See this https://getbootstrap.com/docs/5.2/utilities/display/#hiding-elements

File: index.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    <link rel="stylesheet" href="/css/styles.css" />
+    <script
+      defer
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js"></script>
+    <script defer src="/index.js"></script>
+  </head>
+  <body>
+    <div id="login">
+      <div class="space">
+        <div class="container-xxxl">
```
Comment (sulenchy) [inline index.html]: What class is this `container-xxxl`?

File: package.json [hunk 3]
```diff
@@ -3,7 +3,11 @@
   "version": "0.0.1",
   "description": "A starting point for the CSS Frameworks CA project",
   "scripts": {
-    "test": "echo \"We will learn more about testing in the Workflow course\" && exit 1"
+    "test": "echo \"We will learn more about testing in the Workflow course\" && exit 1",
+    "build": "sass src/scss:dist/css",
+    "watch": "sass --watch sass/styles.scss:css/styles.css",
```
Comment (sulenchy) [inline package.json]: This works, but it won't compile any other scss files apart `styles.scss`.  You can do this instead
`sass --watch sass:css`

**Review by sulenchy — CHANGES_REQUESTED** [review]
@Queenen Run you code in my vcs and I discover this error

```
Error: Can't find stylesheet to import.
   ╷
13 │ @import "../node_modules/bootstrap/scss/bootstrap/scss/carousel.scss";
   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
  sass\styles.scss 13:9  root stylesheet```

This is path is incorrect `@import "../node_modules/bootstrap/scss/bootstrap/scss/carousel.scss";`

File: package.json (hunk 3 above)
Comment (Queenen) [reply package.json]: Alright, I've changed it now!

File: index.html (hunk 2 above)
Comment (Queenen) [reply index.html]: Oops, that's a typo. It's supposed to be "container-xxl" Fixed it. ~

File: css/styles.css (hunk 1 above)
Comment (Queenen) [reply css/styles.css:118]: Ah, I actually learned this as I was working on the project. I forgot to go back and fix it through. Thank you for the note!

**Comment by Queenen** [comment]
This should no longer be an issue. :)

File: css/styles.css (hunk 1 above)
Comment (sulenchy) [reply css/styles.css:118]: @Queenen Here is a reminder that this is yet to be implemented. 

And aside from that, you are writing too much of a custom style. The idea behind this CA is to use strictly Bootstrap with little custom styles.

Note: This is a second chance for you to pass the CA. Failure to implement the changes requested for before the end of today will amount to missing one of the key requirements of the CA. And that is an auto failure.

**Review by sulenchy — CHANGES_REQUESTED** [review]
@Queenen see my comments above

**Pull request closed without merging**